package.json

```diff
@@ -5,7 +5,7 @@
   "description": "Personal Blockchain for Ethereum",
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/trufflesuite/ganache-ui.git"
+    "url": "git+https://github.com/trufflesuite/ganache.git"
```

Comment:
I'm guessing these changes aren't meant to be in here?

Reply:
I've reverted the github link fixes and [put them into a separate PR](https://github.com/trufflesuite/ganache-ui/pull/5123/files). Figured this is already a little messy with all of the style changes from prettier, including more unrelated changes wasn't going to make it easier to follow.